compiler/rustc_mir/src/transform/coverage/debug.rs

```diff
@@ -1,12 +1,294 @@
-use super::graph::CoverageGraph;
+use super::graph::{BasicCoverageBlock, BasicCoverageBlockData, CoverageGraph};
```

Comment:
It would be nice to have a module level doc comment that explains what the debugging options available are and what they do.

Reply:
Done! Check out the new module comments in `debug.rs` when I upload the next set of changes, this afternoon, and let me know what you think.